Resources/views/CRUD/base_list.html.twig

```diff
@@ -159,16 +159,19 @@ file that was distributed with this source code.
                                 </div>
 
 
+                                {# NEXT_MAJOR : remove this assignment #}
+                                {% set export_formats = export_formats|default(admin.getExportFormats) %}
+
                                 <div class="pull-right">
-                                    {% if admin.hasRoute('export') and admin.isGranted('EXPORT') and admin.getExportFormats()|length %}
+                                    {% if admin.hasRoute('export') and admin.isGranted('EXPORT') and export_formats|length %}
```

Comment:
No `admin.getExportFormats()` shouldn't be used anymore, b/c it does not take into account global settings